pkg/model/crd_test.go

```diff
@@ -0,0 +1,78 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReplacePkgName(t *testing.T) {
+	assert := assert.New(t)
+	testCases := []struct {
+		subject         string
+		pkgName         string
+		replacePkgAlias string
+		keepPointer     bool
+		want            string
+	}{
+		{ // most frequent case
+			"*ecr.Repository",
+			"ecr",
+			"svcsdk",
+			true,
+			"*svcsdk.Repository",
+		},
+		{ // don't keep pointer
+			"*ecr.Repository",
+			"ecr",
+			"svcsdk",
+			false,
+			"svcsdk.Repository",
+		},
+		{ // non sdk type
+			"*anotherpackage.Repository",
```

Comment:
let's use a specific real-world example here, yeah? How about `*time.Time`?

Reply:
Yes, it's a good example!